imageprocessor/src/main/java/pl/edu/uj/kimage/processingFlow/FlowStep.java

```diff
@@ -0,0 +1,42 @@
+package pl.edu.uj.kimage.processingFlow;
+
+import pl.edu.uj.kimage.Plugin;
+import pl.edu.uj.kimage.eventbus.Event;
+
+public class FlowStep {
```

Comment:
Outdated code. Final version is that FlowStep is implemented by plugins in order to provide processing logic for given step and dependencies.